pkgs/applications/misc/wtf/default.nix

```diff
@@ -27,6 +31,10 @@ buildGoModule rec {
     runHook postBuild
   '';
 
+  postInstall = ''
+    wrapProgram "$out/bin/wtf" --prefix PATH : "${ncurses.dev}/bin"
```

Comment:
Is this good style or should one better use `stdenv.lib.makeBinPath`?

Reply:
LGTM. makeBinPath is only helpful when you are adding many packages to the PATH.